js/test/units/testunitfmt_be_BY.js

```diff
@@ -0,0 +1,330 @@
+            unit: "square centimeter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY", 
+            autoConvert:true,
+            length:"short"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "2 см²");
+        test.done();
+    },
+    testUnitFormatArea2_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "square centimeter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY", 
+            autoConvert: true,
+            length: "long"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "2 квадратныя сантыметры");
+        test.done();
+    },
+    testUnitFormatArea3_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "hectare",
+            amount: 1000
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY", 
+            autoConvert: true,
+            length: "short"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "10 км²");
+        test.done();
+    },
+    testUnitFormatArea4_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "hectare",
+            amount: 1000
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY", 
+            autoConvert: true,
+            length: "long"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "10 квадратных кіламетраў");
+        test.done();
+    },
+    testUnitFormatFuelConsumption1_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "km/liter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY",
+            autoConvert: true,
+            length: "short"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "2 км/л");
+        test.done();
+    },
+    testUnitFormatFuelConsumption2_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "km/liter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY",
+            autoConvert: true,
+            length: "long"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "2 кіламетра на літр");
+        test.done();
+    },
+    testUnitFormatLength1_be_BY: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "mile",
+            amount: 2000
+        });
+
+        var uf = new UnitFmt({
+            locale: "be-BY",
+            autoConvert: true,
+            length: "short"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "3,21868 Mm");
+        test.done();
+    },
+    testUnitFormatLength2_be_BY: function(test) {
+        test.expect(1);
```

Comment:
Test a different measure that is translated to Belarussian...

Reply:
I updated it